Invoke-HardeningKitty.ps1

```diff
@@ -492,6 +492,22 @@
         Return $AccountSid
     }
 
+    function Write-NotAdminError {
+        [CmdletBinding()]
+        param (
+            [Int]

```

Comment:
The CIS benchmark list for example has an ID like _18.9.108.4.2.2_, therefore we should use `[String]` for the ID as well